src/components/TeamMemberTasks/TeamMemberTasks.jsx

```diff
@@ -377,19 +359,7 @@ const TeamMemberTasks = React.memo(props => {
           taskModalOption={taskModalOption}
         />
       )}
-      <div style={{ maxHeight: '700px', overflow: 'auto' }}>
-        <InfiniteScroll
-          pageStart={0}
-          loadMore={loadFunc}
-          hasMore={hasMore}
-          className="table-container"
-          useWindow={false}
-          loader={
-            <div style={{ textAlign: 'center', marginBottom: '8px' }}>
-              <FontAwesomeIcon icon={faSpinner} pulse /> Loading...
-            </div>
-          }
-        >
+      <div className='table-container'>
           <Table>
             <thead className="pc-component" style={{ position: 'sticky', top: 0 }}>
```

Comment:
I think the in-line style `style={{ position: 'sticky', top: 0 }}>` was added with the infinite scroll, but doesn't seem to be affecting the styles.

Reply:
Good catch! I've removed it now.